src/forms/EditListingAudioForm/EditListingAudioForm.css

```diff
@@ -0,0 +1,175 @@
+@import '../../marketplace.css';
+
+:root {
+  /*
+    These variables are available in global scope through ":root"
+    element (<html> tag). Variables with the same names are going to
+    overwrite each other if CSS Properties' (PostCSS plugin)
+    configuration "preserve: true" is used - meaning that variables
+    are left to CSS bundle. We are planning to enable it in the future
+    since browsers support CSS Properties already.
+ */
+
+  --EditListingAudioForm_imageWidthDesktop: calc(50% - 12px);
+}
+
+.root {
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  flex-shrink: 0;
+}
+
+.imagesField {
+  flex-shrink: 0;
+  margin: 0;
+}
+
+.thumbnail {
+  margin: 24px 0 0 0;
+
+  @media (--viewportLarge) {
+    width: var(--EditListingAudioForm_imageWidthDesktop);
+    margin: 0 24px 24px 0;
+
+    &:nth-child(even) {
+      margin: 0 0 24px 0;
+    }
+  }
+}
+
+.addImageWrapper {
+  float: left;
+  position: relative;
+  width: 100%;
+  margin: 24px 0 0 0;
+  overflow: hidden;
+
+  &::after {
+    content: '.';
+    visibility: hidden;
+    display: block;
+    height: 0;
+    clear: both;
+  }
+
+  @media (--viewportLarge) {
+    width: var(--EditListingAudioForm_imageWidthDesktop);
+    margin: 0 0 24px 0;
+  }
+}
+
+/* Firefox doesn't support image aspect ratio inside flexbox */
```

Comment:
This reminds me. We'll want a ticket to check this on Firefox, Edge, Chrome, and Safari. Good news is that Edge is Chromium now, so basically Chrome with MS skinning. If Andrew makes me support IE or old Edge Ima charge him more. Haha.